src/world_modeling/occupancy_segmentation/src/occupancy_segmentation_node.cpp

```diff
@@ -0,0 +1,56 @@
+#include "occupancy_segmentation_node.hpp"
+#include <iostream>
+#include <memory>
+
+OccupancySegmentationNode::OccupancySegmentationNode() : Node("occupancy_segmentation") {
+  _subscriber = this->create_subscription<sensor_msgs::msg::PointCloud2>(
+      "/velodyne_points", 10,
+      std::bind(&OccupancySegmentationNode::subscription_callback, this, std::placeholders::_1));
+
+  _ground_publisher = this->create_publisher<sensor_msgs::msg::PointCloud2>("/ground_points", 10);
+  _nonground_publisher =
+      this->create_publisher<sensor_msgs::msg::PointCloud2>("/nonground_points", 10);
+}
+
+void OccupancySegmentationNode::subscription_callback(
+    const sensor_msgs::msg::PointCloud2::SharedPtr lidar_cloud) {
+  pcl::PointCloud<PointXYZIRT> temp_cloud;
+  // RCLCPP_INFO(this -> get_logger(), "Header incoming: %s", lidar_cloud ->
```

Comment:
instead of commenting, turn this into a DEBUG message

Reply:
done